galaxy-deploy.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Automated release playbook for Ansible Collections.
+#
+# Originally based on Ericsysmin's 2020 blog post. Meant to be used in a GitHub
+# Actions CI environment.
+#
+# Requires a ANSIBLE_GALAXY_TOKEN secret to be configured on the GitHub repo.
+#
+# Usage:
+#   ansible-playbook -i 'localhost,' galaxy-deploy.yml \
+#   -e "github_tag=${{ github.ref }}"
+
+- hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    # Requires github_tag to be set when calling playbook.
+    release_tag: "{{ github_tag.split('/')[-1] }}"
+
+  pre_tasks:
+    - name: Ensure ANSIBLE_GALAXY_TOKEN is set.
+      fail:
+        msg: A valid ANSIBLE_GALAXY_TOKEN must be set.
+      when: "lookup('env','ANSIBLE_GALAXY_TOKEN') | length == 0"
+
+    - name: Ensure the ~/.ansible directory exists.
+      file:
+        path: ~/.ansible
+        state: directory
+
+    - name: Write the Galaxy token to ~/.ansible/galaxy_token
```

Comment:
Could you confirm that this token won't be shown in any pipeline log?

Reply:
Yes maybe put a no_log here to be sure

Reply:
I did some tests yesterday and today, copy does not output data on -vvv. But I added `no_log` anyway to be on safe side.